packages/api/src/command/medical/document/document-webhook.ts

```diff
@@ -54,12 +54,20 @@ export const processPatientDocumentRequest = async (
     };
     const webhookRequest = await createWebhookRequest({ cxId, type: whType, payload });
     // send it to the customer and update the request status
-    await processRequest(
-      webhookRequest,
-      settings,
-      undefined,
-      patient.data.cxDocumentRequestMetadata
-    );
+    if (!isDisableWH(patient.data.cxDocumentRequestMetadata)) {
+      await processRequest(
+        webhookRequest,
+        settings,
+        undefined,
+        patient.data.cxDocumentRequestMetadata
+      );
+    } else {
+      const status = "success";
+      await updateWebhookRequestStatus({
+        id: webhookRequest.id,
+        status,
+      });
```

Comment:
We talked about this on the huddle, we could create it already with the status "success", there's no need for an update right after a create, right?

Reply:
I thought we said the opposite but will change.